src/components/Avatar.js

```diff
@@ -2,8 +2,12 @@
 // returns an img element with className 'Avatar'
 // img element should have src and alt attributes
 
-export default function Avatar(props) {    
-    const { person } = props;
+export default function Avatar({ person }) {
+    // const { person } = props;
```

Comment:
Only add meaningful instructions or reminders as a comment, you can delete this one